styles/colors-electro.less

```diff
@@ -81,10 +83,10 @@
 
 // Checkbox
 // ---------------------------------------
-@agate-checkbox-text-color: @agate-white;
-@agate-checkbox-bg-color: @agate-accent;
+@agate-checkbox-text-color: @agate-component-text-color;
+@agate-checkbox-bg-color: @agate-component-bg-color;
 @agate-checkbox-focus-bg-image: none;
-@agate-checkbox-focus-bg-color: @agate-background-focus;
+@agate-checkbox-focus-bg-color: @agate-focus-bg-color;
```

Comment:
Could you please check `@agate-radioitem-icon-focus-bg-color` in line 184?

Reply:
it looks logical to aso update @agate-radioitem-icon-focus-bg-color. Thank you!